makefile

```diff
@@ -27,15 +27,22 @@ build:
 
 ########## LAUNCH ##########
 
-run: check-valid-service build down
+run: check-valid-service
 	$(COMPOSE) up -d $(shell $(COMPOSE) config --services | grep -E "($(SERVICE)|common)$$")
 
-dev:
+dev: build down
+	$(MAKE) run SERVICE=resource
 	$(MAKE) run SERVICE=dev
 
-prod:
+prod: build down
 	$(MAKE) run SERVICE=prod
 
+res: resources
+resource: build down
+resources: build down
+	$(MAKE) run SERVICE=resource
```

Comment:
What is the difference in use between these, if any?

Reply:
It's just an alias, some people might not remember if it's `make resource` or `make resources`. It's just better to make both forms of the command run the same thing to reduce frustration caused by such a small problem.

Reply:
I didn't realize at the time that line 43 runs after any of the previous 3 commands - it's obvious now that they're the same 😅